### ISSUE TITLE
Rubber Trees

### ISSUE THREAD

**Issue by fsedarkalex** [Kasax007/Dreamcraft-Development-#108]
Rubber trees are buggy on servers: When they grow, it requires a restart of the game client for them to actually appear.
Same goes for when they are harvested

**Comment by DarkCrystalll** [comment]
There are many situations in the server that are like this, caused by the server and client being out of sync. I saw a mod in the mod list that fixes this issue, which may be the cause of this mod

**Comment by fsedarkalex** [comment]
Yes while playing further we noticed many block entities are having issues syncing their changes after initial load. Cables, rubber trees, machines, ...

And we noticed, right-clicking the blocks with bare hands and some tools causes them to update

**Comment by fsedarkalex** [comment]
> I saw a mod in the mod list that fixes this issue, which may be the cause of this mod

Could you say which one? Would like to tinker a bit.

**Comment by DarkCrystalll** [comment]
AntiGhost,it said it can fix the problem,but it don't work.maybe close this mod can fix this problem.

**Comment by DarkCrystalll** [comment]
I'm not sure

**Comment by fsedarkalex** [comment]
I didn't try it yet but this mod aims for something else.
